src/main/java/OGF.java

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+public class OGF {
+    private static final int MAX_ITEMS = 100;
+    private static int numItem;
+    private static final Task[] tasks = new Task[MAX_ITEMS];
+
+    private static boolean isRunning = true;
+    private static void printBreakLine(){
+        System.out.println(("____________________________________________________________"));
+    }
+
+    private static void printTaskAdded(Task task, int taskIndex){
+        System.out.println("Alright, adding this task to the list: ");
+        System.out.println(task);
+        System.out.printf("You have %d tasks in the list.%n", taskIndex+1);
+        printBreakLine();
+    }
+
+    private static boolean handleInput (String input) throws OGFException{
+        switch (input.split(" ")[0]) {
+            case ("bye"):
+                System.out.println("Bye bye now!");
+                return (false);
+            case ("list"):
+                System.out.println("Here are your tasks for today: ");
+                for (int i = 0; i < numItem; i++) {
+                    System.out.print(i + 1 + ". ");
+                    System.out.println(tasks[i]);
+                }
+                printBreakLine();
+                break;
+            case ("mark"):
+                //Fallthrough
+            case ("unmark"):
+                if (!input.contains(" ")){
+                    throw new OGFException("did not indicate task to mark/unmark");
+                }
+
+                int taskNo = Integer.parseInt(input.split(" ")[1]) - 1;
+                if (taskNo < 0 || taskNo > numItem-1){
+                    throw new OGFException("tried to mark/unmark task not in list");
+                }
+                if (input.split(" ")[0].equals("mark")) {
+                    System.out.println("Good Job! I'm setting this task as done: ");
+                    tasks[taskNo].setDone(true);
+                } else {
+                    System.out.println("Oop! I'm setting this task as undone: ");
+                    tasks[taskNo].setDone(false);
+                }
+                System.out.println(tasks[taskNo]);
+                printBreakLine();
+                break;
+            case ("todo"):
+                if (!input.contains(" ") || input.indexOf(" ") == input.length()-1){
```

Comment:
This is a complex statement, so try to break it down into two boolean variables which handle each sub-expression.